iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/receiver/IoTDBFileReceiver.java

```diff
@@ -71,6 +73,10 @@ public abstract class IoTDBFileReceiver implements IoTDBReceiver {
   protected boolean shouldConvertDataTypeOnTypeMismatch =
       CONNECTOR_EXCEPTION_DATA_CONVERT_ON_TYPE_MISMATCH_DEFAULT_VALUE;
 
+  // Used to determine current strategy is sync or async
+  protected static final AtomicBoolean IS_USING_ASYNC_LOAD_TS_FILE_STRATEGY =
```

Comment:
Better not use static here

Reply:
If static is not used in that place, if the user switches the loading method, there may be problems. The test found that the value is both false and true during synchronization. @SteveYurongSu